test/_SireWrappers/test_system.py

```diff
@@ -144,3 +144,17 @@ def test_contains_combine(system):
     assert m0 in s1
     assert m1 in s0
     assert m1 in s1
+
+def test_get_atom(system):
```

Comment:
Can we add unit tests for calling `getAtom()`, modifying the system, and calling `getAtom()` again, just to make sure it works as expected?

Reply:
I've updated the test so that we delete some molecules from the system, then make sure that `getAtom()` still works, i.e. it maps to the new molecule positions, not the old ones.